src/utils/noting/find-entire-note.js

```diff
@@ -1,7 +1,7 @@
 var isVFocus = require('../vfocus/is-vfocus');
 var isWithinNote = require('./is-within-note');
-var isNote = require('./is-note');
-var findFirstNote = require('./find-first-note');
+var isNote = require('./is-note-segment');
+var findFirstNote = require('./find-first-note-segment');
```

Comment:
There are some more similar ones elsewhere where the variable is lacking `Segment` at the end of the name.